ui/app/src/components/MyRecentActivityDashlet/MyRecentActivityDashlet.tsx

```diff
@@ -179,12 +180,13 @@ export function MyRecentActivityDashlet(props: MyRecentActivityDashletProps) {
     const events = [deleteContentEvent.type, workflowEvent.type, publishEvent.type, contentEvent.type];
     const hostToHost$ = getHostToHostBus();
     const subscription = hostToHost$.pipe(filter((e) => events.includes(e.type))).subscribe(({ type, payload }) => {
+      dispatch(fetchSandboxItems({ paths: selectedPaths }));
       loadPage(getCurrentPage(offset, limit), true);
     });
     return () => {
       subscription.unsubscribe();
     };
-  }, [limit, offset, loadPage, username]);
+  }, [limit, offset, loadPage, username, dispatch, selectedPaths]);
```

Comment:
@jvega190 Did the linter not complain about this or you just missed the warning?

Reply:
I saw no warnings, do you mean about `fetchSandboxItems`?

Reply:
The warning about the effect not having all dependencies listed (the cause of the bug)

Reply:
Never mind I just noticed that was a new line too